config/container_resolvers_conf.xml.sample

```diff
@@ -0,0 +1,35 @@
+<containers_resolvers>
+  <explicit />
+  <!-- explicit: resolves container URI for a job through explict container
+       tags in the tool XML wrapper. -->
+
+  <!-- All mulled flavors below only work if enable_beta_mulled_containers is 
+       set to true in the galaxy.yaml config file.
+
+  <!-- <cached_mulled /> -->
+  <!-- cached_mulled: resolves container URI through bioconda -> mulled
+       automatic mapping, preferring cached containers in the accessible docker
+       engine. Requires docker engine. -->
+
+  <!-- <cached_mulled_singularity /> -->
+  <!-- cached_mulled_singularity: resolves container URI through
+       bioconda -> mulled automatic mapping, preferring cached singularity
+       containers accessible to local singularity binaries. Only works with
+       enable_beta_mulled_containers set to true in the galaxy.yaml config file.
+       -->
+
+  <!-- <mulled /> -->
+  <!-- mulled: resolves container URI through bioconda -> mulled automatic
+       mapping. Doesn't require docker or singularity, and should be the method
```

Comment:
`Doesn't require docker or singularity` this is a little bit confusing, imho.

Reply:
Will rephrase.